logrecorder-log4j/src/main/kotlin/io/github/logrecorder/log4j/kotest/Log4jRecorderKotestExtension.kt

```diff
@@ -0,0 +1,47 @@
+package io.github.logrecorder.log4j.kotest
+
+import io.github.logrecorder.api.LogRecord
+import io.github.logrecorder.common.kotest.AbstractLogRecorderKotestExtension
+import io.github.logrecorder.log4j.Log4jLogRecord
+import io.github.logrecorder.log4j.Log4jLogRecorder
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import kotlin.reflect.KClass
+
+/**
+ * Convenience Method to instantiate [Log4jRecorderKotestExtension]
+ * This extension will record the loggers specified by the parameters save the [LogRecord] into the CoroutineContext
+ * for retrieval by logRecord
+ *
+ * Recording a logger will set its log level to [org.apache.logging.log4j.Level.ALL] for the duration of the test.
+ * After the test was executed, the log level will be restored to whatever it was before.
+ *
+ * @param classes [KClass] whose name should be used to identify loggers to record.
+ * @param names String names of the [Logger] instances to record
+ * @see Log4jRecorderKotestExtension
+ * @since 2.4
+ */
+@PublishedApi
+internal fun recordLogs(
```

Comment:
if they are all `internal`, nobody can use them - how do I use the extension?

Reply:
The @PublishedApi makes it usable